pkg/bosh/manifest/dns.go

```diff
@@ -207,12 +237,43 @@ func (dns *boshDomainNameService) Reconcile(ctx context.Context, namespace strin
 	return nil
 }
 
+func (dns *boshDomainNameService) createCorefile(namespace string) string {
+	rewrites := make([]string, 0)
+	for _, alias := range dns.Aliases {
+		for _, target := range alias.Targets {
+			from := alias.Domain
+			if target.Query == "_" {
+				from = strings.Replace(from, "_", target.InstanceGroup, 1)
+			}
+			to := fmt.Sprintf("%s.%s.svc.cluster.local", dns.HeadlessServiceName(target.InstanceGroup), namespace)
```

Comment:
`.cluster.local` isn't actually a constant, unfortunately.  Some clusters have setups where that's incorrect… not sure how to get the correct value though.

Reply:
You are right. The cluster domain is configurable. I'll expose a flag on the cf-operator to allow passing it. It should also be available from the Helm values.

Reply:
Done in https://github.com/cloudfoundry-incubator/cf-operator/pull/642/commits/797674cd9d380dde4cbfce4a2b2f9764b93a2dd5.